Updated documentation

jaswantcoder opened the pull request:
*Issue # updated documentation*

Commit:
udpated example readme

Commit:
increment version

Commit:
udpate example readme

debanshuk commented on EXAMPLE.md:
```diff
@@ -6,31 +8,40 @@ import {
   credentials: {
     accessKeyId: 'EXAMPLEACCESSKEY',
     secretAccessKey: 'examplesecretkey/K',
   },
```
Nit: We can skip this part from the example, as we don't want to promote usage of IAM users.

jaswantcoder replied:
removed

debanshuk commented on EXAMPLE.md:
```diff
@@ -6,31 +8,40 @@ import {
-    kid: 'arn:aws:kms:eu-west-1:123456123456:alias/ExampleSigningKey',
-    crit: ['exp'],
+    alg: 'SIGNING_ALGORITHM',
+    kid: 'KMS_SIGNING_KEY_ID',
```
At some place in this file we should mention that 'KMS_SIGNING_KEY_ID' (and other KMS key IDs) can be key-id, key-id-arn, key-alias or key-alias-arn, based upon the user's needs.

jaswantcoder replied:
added a Note at the end of file

debanshuk commented on EXAMPLE.md:
```diff
@@ -43,22 +54,31 @@ const jws: Promise<string> = signer
+### Encryption
+
+1. Prepare encryption options with format as `compact`, header fields and cek as KMSSymmetricCEK object.
+2. Prepare encryption recipient with reference as `false`, and key as KMSSymmetricKey object. Use same kms key to create KMSSymmetricCEK and KMSSymmetricKey.
```
Nit: 'kms' -> 'KMS'.

jaswantcoder replied:
done

mjsheikh commented:
Please enable atleast the following branch protection rules for the `main` branch with the following settings:
1. Require a pull request before merging
a. Require approvals
b. Dismiss stale pull request approvals when new commits are pushed

Commit:
address comments

debanshuk reviewed (commented):
It seems there a 4 commits in this PR with same/similar commit messages. Could you please merge these commits to a single commit?

jaswantcoder commented:
> It seems there a 4 commits in this PR with same/similar commit messages. Could you please merge these commits to a single commit?

Yes, squashed commits into one and merged. Deleted branch
